python/tests/test_framework/test_configs/slate.json

```diff
@@ -0,0 +1,130 @@
+[
+    {
+        "test_name": "slates",
+        "data_func": {
+            "name": "generate_slate_data",
+            "params": {
+                "num_examples": 1000,
+                "num_action": [
```

Comment:
looks like there is typo: num_action instead of num_actions and we are not detecting it since we are relying on *args call in test_core. Probably better to switch to **kwargs one and enforce named arguments always